data_safe_haven/infrastructure/programs/sre/software_repositories.py

```diff
@@ -124,31 +124,25 @@ def __init__(
         )
 
         # Upload Nexus allowlists
-        cran_reader = FileReader(
-            resources_path / "software_repositories" / "allowlists" / "cran.allowlist"
-        )
         FileShareFile(
             f"{self._name}_file_share_cran_allowlist",
             FileShareFileProps(
-                destination_path=cran_reader.name,
+                destination_path="cran.allowlist",
                 share_name=file_share_nexus_allowlists.name,
-                file_contents=cran_reader.file_contents(),
+                file_contents="",
                 storage_account_key=props.storage_account_key,
                 storage_account_name=props.storage_account_name,
             ),
             opts=ResourceOptions.merge(
                 child_opts, ResourceOptions(parent=file_share_nexus)
             ),
```

Comment:
```suggestion
            opts=ResourceOptions.merge(
                child_opts, ResourceOptions(
                parent=file_share_nexus,
                ignore_changes=["file_contents"],
            )
                )
            ),
```

Just to be on the safe side :smile: